src/main/resources/application.properties

```diff
@@ -1,3 +1,7 @@
 quarkus.container-image.build=true
 
-quarkus.http.port=${PORT:8080}
\ No newline at end of file
+quarkus.http.port=${PORT:8080}
+
+quarkus.hibernate-orm.database.generation=none
+#%dev.quarkus.hibernate-orm.sql-load-script=scripts/load-data.sql
+quarkus.datasource.devservices.init-script-path=scripts/schema-load-data.sql
```

Comment:
Can you add %dev to the front of these configs and delete the commented out one?